srcpkgs/glibc/template

```diff
@@ -195,7 +195,6 @@ glibc-devel_package() {
 		vmove usr/include
 		vmove "usr/lib/*.a"
 		vmove "usr/lib/*.o"
-		vmove usr/share/info
```

Comment:
This change is still valid for the bootstrap case.

Reply:
@pullmoll this one is valid because if there's no texinfo the installation will fail.

This needs to be handled for the `-z $CHROOT_READY` case.

Reply:
Ok, I'm currently trying a bootstrap with x86_64 glibc and when I see it, push the fix.

Reply:
I think this would be ok:
```
                if [ -d "${DESTDIR}/usr/share/info" ]; then
                        vmove usr/share/info
                fi
```
Thus if you actually had texinfo installed in the host during a bootstrap, you can have the info files for glibc installed. Or should we rely solely on `$CHROOT_READY`?

Reply:
much better!

Reply:
That's a great solution, @pullmoll